web-client/src/presenter/presenter.ts

```diff
@@ -1646,6 +1649,59 @@ export const presenterSequences = {
     validateUserContactSequence as unknown as Function,
 };
 
+function wrapActionsWithPerformanceLogic(structure) {
```

Comment:
I'm not sure I fully understand the behavior of nested Sequences. For instance, in the following screen shot (going to case 107-19 and editing an existing Status Report Order), I get two different time values for the same sequence `'getStatusReportOrderPdfUrlAction2'`, which should have only been run once:

<img width="614" alt="Screenshot 2024-08-15 at 3 30 06 PM" src="https://github.com/user-attachments/assets/fc3464d5-d6b3-442b-88c8-ac7302c3ad04">

I'm not sure how much this will skew other sequences.

Reply:
We are! It looks like the logging might help us find instances where we are making redundant calls--it's already helped once!